test/unit/lib/formatting.cpp

```diff
@@ -96,6 +96,21 @@ TEST(FormatterTest, ObjectAddresses) {
     );
 }
 
+TEST(FormatterTest, NoAddresses) {
+    auto formatter = cpptrace::formatter{}
+        .addresses(cpptrace::formatter::address_mode::none);
+    auto res = split(formatter.format(make_test_stacktrace()), "\n");
+    EXPECT_THAT(
+        res,
+        ElementsAre(
+            "Stack trace (most recent call first):",
+            "#0  in foo() at foo.cpp:20:30",
```

Comment:
I think we'll want this to format as `#0 in foo() at foo.cpp:20:30`. This makes the change in formatting.cpp slightly more annoying but not too bad.
(This is also important for `cpptrace::stacktrace_frame::to_string` etc, at the moment that'll return ` in foo() at foo.cpp:20:30`)